app/javascript/components/forms/DeletePresentationForm.jsx

```diff
@@ -12,18 +12,18 @@ import useDeletePresentation from '../../hooks/mutations/rooms/useDeletePresenta
 export default function DeletePresentationForm({ handleClose }) {
   const methods = useForm();
   const { friendlyId } = useParams();
-  const { handleDeletePresentation, isLoading } = useDeletePresentation(friendlyId);
+  const deletePresentation = useDeletePresentation(friendlyId);
   return (
     <>
       <p className="text-center"> Are you sure you want to delete this presentation?</p>
-      <Form methods={methods} onSubmit={handleDeletePresentation}>
+      <Form methods={methods} onSubmit={deletePresentation.mutate}>
```

Comment:
If we're gonna use the `mutate` directly I think that we need to:
1. Roll it out everywhere else for consistency.
2. Prepare the data on the `axios` callback level instead on the mutation wrapper that we're doing now, i.e. to send room data from the form it needs to be encapsulated inside of a room property of an object so the `axios` callback should be `(room) => axios.post('endpoint',{ room })`. 
3. Use `mutateAsync` for form when using the `isSubmitting` form state or use `mutate` but use the mutation state instead.

Reply:
I don't understand #3. We should use mutateAsync for forms, and mutate for other mutations?

Reply:
> > Prepare the data on the axios callback level instead of doing that on the mutation wrapper function (that's what we're doing now), i.e. to send room data from the form it needs to be encapsulated inside of a room property of an object so the axios callback should be (room) => axios.post('endpoint',{ room }).
> 
> I'm not sure I understand
> 
Check the changes made to this mutation and how in order to remove the need for the handler wrapping function to encapsulate the room data I'm doing that on the mutation function (axios funtion).
https://github.com/bigbluebutton/greenlight/pull/3618/files#diff-58e8c46468b642313d66ecbee89bb9adc3ef4b4a672495ff26506000267db71cR9

Reply:
Sorry for the late replies guys 🙏